Minor changes for Travis CI

solflare opened the pull request:
In the pull here: https://github.com/iFixit/pdf.js/pull/14 we change the url from the ssh version to https so Travis CI would work. However in our private ci runs we verify that there are _no_ https git submodule urls. This reverts the change in the pull above and adds a change to the travis.yml file that will convert the url from ssh to https.

davidrans commented:
Makes sense to me. Might be nice to add a comment explaining why it's necessary.
CR ☕️

davidrans commented:
CR ☕️

archive64 commented on .travis.yml:
```diff
@@ -1,3 +1,11 @@
+  submodules: false
+# Use sed to replace the SSH URL with the public URL, then initialize submodules
+before_install:
+  - sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
```
I think there is a [git command to do this](http://stackoverflow.com/a/928539).

Something like: `git config submodule.fonttools.url https://github.com/behdad/fonttools.git`

solflare commented:
dev_block :hand:

solflare commented:
un_dev_block :moon: even though that doesn't really work here D:

davidrans commented:
CR ☕️

solflare commented:
I squashed the commits so it'll be easier to rebase onto mozilla's master in the future.

solflare committed:
Use ssh url for fonttools, Setup travis ci to use https

- We want to use the ssh url for git modules so we need to have travis
change the submodule url before_install using `git config`.

- For now we're only going to run the lint test until we can update our
fork or make the necessary changes to use tests that have been added to
the main repo since we forked it.

archive64 commented:
CR 👍

davidrans commented:
CR ☕️

solflare commented:
Since CI passed I'm going to merge this.

solflare commented:
agh I lost a change in the .gitmodules X( Opening a new pull
